lib/team.rb

```diff
@@ -0,0 +1,31 @@
+class Team
+  attr_reader :country, :players, :eliminated, :player_position,
+              :add_player, :players_by_position
+
+  def initialize(country)
+    @country = country
+    @players = []
+    @player_position = []
+    @eliminated = false
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def eliminated
+    @eliminated = true
+  end
+
+  def add_player(player_input)
+    @players << player_input
+  end
+
+  def players_by_position(position_input)
```

Comment:
For players by position, you want to `find_by` the position that you input to return the player objects.  It might look something like this:
```ruby
def players_by_position(position_input)
  @players.find_by do |player|
    player.position == position_input
  end
end
```